test/acceptance/specs/RevokeApplicationAuthoritySpec.scala

```diff
@@ -52,7 +53,7 @@ class RevokeApplicationAuthoritySpec extends BaseSpec with NavigationSugar {
       clickOnSubmit()
 
       on(PermissionWithdrawnPage(app.application.id))
-      verifyText(withdrawnMessageText, s"${app.application.name} no longer has authority to interact with HMRC on your behalf.")
+      verifyText(withdrawnMessageText, "This application no longer has authority to interact with HMRC on your behalf.")
```

Comment:
Yes, we've agreed that changing the copy is the preferred solution (rather than serving different content under the same url as would be the case if we would not make a redirect).